### ISSUE TITLE
Great work! A few questions to ask

### ISSUE THREAD

**Issue by XqZeppelinhead0702** [IntMeGroup/AIGV-Assessor#1]
Hi, really appreciate your marvellous work and significant contribution to the open source community! However, I still have some questions to ask:
1. Does AIGV-Assessor also do well in longer videos? Considering most videos of the AIGVQA-DB are not very long in duration, it's intriguing how it works on longer videos such as multi-scene videos.
2. How could I do to evaluate the quality of some videos out of AIGVQA-DB test set? Do you have any plan to release the pretrained model weights or  sample inference code? Or should I train the model myself and use the trained checkpoints to test?
3. How is the training cost of AIGV-Assessor? Did you gauge the GPU hours?  I wonder how long it will take to train the model completely on 8 A100 GPUs for example.
